src/Constant.tsx

```diff
@@ -333,113 +294,971 @@ export const getIndicatorsTree = (): TIndicatorsTree[] => [
+            defaultValue: 8,
+            category: 'inputs',
+            type: 'number',
+        },
+        {
+            path: 'teethPeriod',
+            title: t.translate('Teeth Period'),
+            defaultValue: 8,
+            category: 'inputs',
+            type: 'number',
+        },
+        {
+            path: 'teethOffset',
+            title: t.translate('Teeth Offset'),
+            defaultValue: 5,
+            category: 'inputs',
+            type: 'number',
+        },
+        {
+            path: 'lipsPeriod',
+            title: t.translate('Lips Period'),
+            defaultValue: 5,
+            category: 'inputs',
+            type: 'number',
+        },
+        {
+            path: 'lipsOffset',
+            title: t.translate('Lips Offset'),
+            defaultValue: 3,
+            category: 'inputs',
+            type: 'number',
+        },
+        {
+            path: 'showFractal',
+            title: t.translate('Show Fractals'),
+            type: 'switch',
+            category: 'parameters',
+            defaultValue: false,
+        },
+    ],
+};
+
+const FractalChaosBandIndicatorConfig: TDefaultIndicatorConfig = {
+    parameters: [
+        {
+            path: 'channelFill',
+            title: t.translate('Channel Fill'),
+            type: 'switch',
+            category: 'parameters',
+            defaultValue: true,
+        },
+    ],
+};
+
+export const DefaultIndicatorConfigs: TDefaultIndicatorConfigs = {
+    RSI: RSIIndicatorConfig,
+    adx: ADXIndicatorConfig,
+    AwesomeOscillator: AwesomeOscillatorIndicatorConfig,
+    dpo: DPOIndicatorConfig,
+    gator: GatorIndicatorConfig,
+    macd: MACDIndicatorConfig,
+    ROC: ROCIndicatorConfig,
+    StochasticOscillator: StochasticOscillatorIndicatorConfig,
+    SMI: SMIIndicatorConfig,
+    WilliamsR: WilliamsRIndicatorConfig,
+    Aroon: AroonIndicatorConfig,
+    commodity_channel_index: CCIIndicatorConfig,
+    ichimoku: IchimokuIndicatorConfig,
+    ParabolicSAR: ParabolicSARIndicatorConfig,
+    zigzag: ZigzagIndicatorConfig,
+    bollinger_bands: BollingerBandsIndicatorConfig,
+    donchian_channel: DonchianChannelIndicatorConfig,
+    moving_average: MAIndicatorConfig,
+    moving_envelope_average: MAEnvIndicatorConfig,
+    rainbow: RainbowIndicatorConfig,
+    alligator: AlligatorIndicatorConfig,
+    fcb: FractalChaosBandIndicatorConfig,
 };
 
 export const ChartTypes = [
-    { id: 'mountain', text: t.translate('Area'), candleOnly: false, icon: TypeAreaIcon },
-    { id: 'candle', text: t.translate('Candle'), candleOnly: true, icon: TypeCandleIcon },
-    { id: 'hollow_candle', text: t.translate('Hollow'), candleOnly: true, icon: TypeHollowIcon },
-    { id: 'colored_bar', text: t.translate('OHLC'), candleOnly: true, icon: TypeOhlcIcon },
+    { id: 'line', text: t.translate('Area'), candleOnly: false, icon: TypeAreaIcon },
+    { id: 'candles', text: t.translate('Candle'), candleOnly: true, icon: TypeCandleIcon },
```

Comment:
@balakrishna-deriv what's the main reason to rename the ids? is it only for consistency between smartcharts and flutter-charts?

Reply:
Yes, to make it consistent between smartcharts and flutter-charts